contentcuration/contentcuration/management/commands/mark_incomplete.py

```diff
@@ -0,0 +1,47 @@
+from django.core.management.base import BaseCommand
+from django.db.models import Exists
+from django.db.models import OuterRef
+from django.db.models import Q
+from le_utils.constants import content_kinds
+from le_utils.constants import exercises
+
+from contentcuration.models import AssessmentItem
+from contentcuration.models import ContentNode
+from contentcuration.models import File
+
+
+class Command(BaseCommand):
+    def handle(self, *args, **options):
+        exercise_check_query = AssessmentItem.objects.filter(contentnode=OuterRef('id')) \
```

Comment:
Another thing to check could be if a single selection question has more correct answers. Same applies to true/false question. And maybe some other cases (I was just reading[ this frontend test suite](https://github.com/learningequality/studio/blob/develop/contentcuration/contentcuration/frontend/channelEdit/__tests__/utils.spec.js#L128)).
But I am not sure it's possible to have such errors in database (I suppose that the old frontend prevented from saving those), and going into such subtleties might not even be needed for this command I suppose (in the worst case, it will be validated again on frontend after the load).

Reply:
I'm hoping that people haven't been doing that and we're properly blocking that on the frontend. Unfortunately, I don't see a good way of handling this given that the answers are just strings

Reply:
Yeah, doesn't seem to be such a problem to me too.
Besides that I believe our frontend, users and https://media.istockphoto.com/vectors/hope-changes-everything-card-vector-id616113880 :))))